features/users/profile.feature

```diff
@@ -53,3 +53,15 @@ Feature: As a user of the site
       | hello world   |
       | hello jupiter |
 
+  Scenario: Having karma count on users profile page
+    Given user "Bob" have karma:
+      | total | hangouts_attended_with_more_than_one_participant |
+      | 20    | 20                                               |
+    When I click on the avatar for "Bob"
+    Then I should be on the "user profile" page for "Bob"
+    And I should see "20"
+  
+  Scenario: Having karma count as zero on users profile page with zero hangouts_attended_with_more_than_one_participant
+    When I click on the avatar for "Alice"
+    Then I should be on the "user profile" page for "Alice"
+    And I should see "0"
```

Comment:
we might want to check for the css tag as well as just the number itself ...

Reply:
I have checked for CSS tags as well as the value inside those CSS tags. Thanks, @tansaku for the suggestion.